.travis.yml

```diff
@@ -0,0 +1,19 @@
+dist: xenial
+language: go
+go:
+  - 1.14.1
+env:
+  - REALESE_BRANCHES="release-v[0-9]+\.[0-9]+\.x" RELEASE_TAGS="v[0-9]+\.[0-9]+\.[0-9]+(-rc[0-9]+)?" BUILD_OS="$([ $TRAVIS_EVENT_TYPE == cron ] || [[ $TRAVIS_BRANCH =~ $RELEASE_BRANCHES ]] || [[ $TRAVIS_TAG =~ $RELEASE_TAGS ]] && echo darwin) linux" BUILD_ARCH=amd64 GATE_API_BRANCH=release-1.20.x SEND_COVERITY=send
```

Comment:
Can you use here multiline syntax for example with `|` ?

Reply:
Improved readability but I don't see how I could make declaration of BUILD_ARCH more clear.